README.md

```diff
@@ -460,8 +460,47 @@ HTTP 204 No content
 In this first release, where we have no concept of an organisation, the following endpoints have been hardcoded to return
 certain values.
 
+* [List my organisations](#list-my-organisations)
 * [List organisation assessments](#list-organisation-assessments)
 
+## List my organisations
+
+```
+GET /organisations/
+```
+
+List all organisations the logged in user is part of
+
+ℹ️ porting notes: replaces previous `assessment/list` route, passing `orgid`.
```

Comment:
`assessment/list` with `orgid` lists *assessments* within an organisation, not my organisations

Reply:
`assessment/getorganisations`

Reply:
Fixed! Thanks!